www/include/Administration/parameters/DB-Func.php

```diff
@@ -1051,7 +1051,7 @@ function updateRemoteAccessCredentials($db, $form, $centreonEncryption): void
     unset($ret['apiCredentials']);
 
     //convert values
-    $ret['apiSelfSignedCertificate'] = $ret['apiSelfSignedCertificate'] == 1 ? 'yes' : 'no';
+    $ret['apiPeerValidation'] = (int) $ret['apiPeerValidation'] === 1 ? 'no' : 'yes';
```

Comment:
`1` means `no`?

Reply:
it's odd indeed. When checked the value is '1' for -> 
![image](https://user-images.githubusercontent.com/34628915/95475002-0dfec480-0986-11eb-8456-4a606fba507a.png)

But when checked, "allow self signed certificate" means do not check "peer validation"
so yes, 1 means no in DB :)

Reply:
ok :)
so the name of the form input is not good

Reply:
let's update this later